services/data/db_utils.py

```diff
@@ -14,6 +15,28 @@
 def aiopg_exception_handling(exception):
     err_msg = str(exception)
     body = {"err_msg": err_msg}
+    if isinstance(exception, asyncio.TimeoutError):
+        body = {
+            "err_msg": {
+                "type": "timeout error",
+            }
+        }
+    elif psycopg2.Error in type(exception).__bases__:
```

Comment:
what's the difference between this and `isinstance(exception, psycopg2.Error)` ?

Reply:
None actually. I can change that.